extensions/GoogleAnalytics/web/js/dnt-helper.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+/**
+ * Returns true or false based on whether doNotTack is enabled. It also takes into account the
```

Comment:
Typo: doNotTrack

Reply:
This file is just a copy from [Bedrock](https://github.com/mozilla/bedrock/blob/master/media/js/base/dnt-helper.js) so I'll fix it separately 😆

Reply:
Fixed.